Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -39,6 +41,7 @@ import SwiftUI
 ///     // Action that should be performed on pressing the "Continue" button ...
 /// }
 /// ```
+///
```

Comment:
I would suggest to remove any empty lines at the end here.

The addition of the comment above is great but a bit small for a documentation PR.
I would suggest to add some example and explanation on how to adjust the onboarding view with e.g. a custom header or a custom main view or footer. Those are questions that are asked a lot and might be great to be addressed.

Reply:
Thank you so much for your feedback! I have adjusted the documentation to add an example with a custom view and details about that. Please let me know if you have any comments on this version!